app/build.gradle

```diff
@@ -64,7 +64,8 @@ dependencies {
       'com.google.dagger:dagger:2.24',
       "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
       "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.2.1",
-      "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.2.1"
+      "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.2.1",
```

Comment:
Super nit: please use single quote strings unless you're including a variable. Probably ditto for the kotlinx-coroutines-core bit above.

Reply:
done